src/main/resources/plugins/pmd-ruleset.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset name="dal_avdevices_power_middleatlantic PMD ruleset"

```

Comment:
Why does it have middle atlantic in the name?

Reply:
This configuration does not change so the name change was missed, fixing